packages/cadl-ranch-specs/http/azure/client-generator-core/internal/main.tsp

```diff
@@ -20,6 +20,12 @@ model InternalModel {
   name: string;
 }
 
+@doc("This is a model only used by internal operation. Also, it is decorated with @include. It should be generated and exported.")
+@global.Azure.ClientGenerator.Core.include
+model InternaIncludelModel {
+  name: string;
```

Comment:
Should we add a nested model to also test transitivity?

Or, if we don't sure how transitivity works for `@include`, I am find to leave as this.

Reply:
I think transitivity should be the work of `isInclude` (though it seems current TCGC has not implemented it yet). I've created an [issue](https://github.com/Azure/typespec-azure/issues/3264) to track..

Reply:
I think we should figure out this issue now.